pom.xml

```diff
@@ -53,7 +53,7 @@
         <module>aws-sdk-java</module>
         <module>core</module>
         <module>services</module>
-        <module>services-hll/dynamodb-enhanced</module>
+        <module>services-custom/dynamodb-enhanced</module>
```

Comment:
How about `services-enhanced`?

Reply:
services-enhanced feels like it'd just be for `ServiceNameEnhancedClient`s. If we decide that `TransferManager` should go in a separate module, it wouldn't make sense here, imo.

Reply:
Discussed offline. Yeah, I'm fine with this since this is not customer facing and we can change it later if we want.